lmod_proxy/edx_grades/actions.py

```diff
@@ -30,6 +30,16 @@ def post_grades(gradebook, form):
     csv_file.seek(0)
     results = None
     try:
+        # Create assignment to explicitly set max points.
```

Comment:
Unfortunately, this "table" is not normalized.  ``max_points`` are an attribute of the assignment (the table), not the student (each row).  It is the same for each row.